zips/zip-0230.rst

```diff
@@ -222,86 +203,111 @@ Transaction Format
 
```

Comment:
At line 191, change ``SpendDescriptionV6`` to ``SpendDescriptionV5``, and add an extra bullet point:
```rst
* There is no ``SpendDescriptionV6`` because the format of Sapling Spend descriptions
  has not changed from SpendDescriptionV5`` [#zip-0225]_.
```